lib/tasks/after_tasks.rake

```diff
@@ -141,42 +141,6 @@ namespace :After do
     puts("Added default rating to works: #{updated_works}") && STDOUT.flush
   end
 
-  desc "Fix pseuds with invalid icon data"
-  task(fix_invalid_pseud_icon_data: :environment) do
```

Comment:
Does this need to be run one last time before being deleted or are we good already?